src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -109,7 +117,7 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
               <div className="profile-pic">
                 {/* @debt Refactor this to use our useImage hook? Or just UserAvatar / UserProfilePicture directly? */}
                 <img
-                  src={selectedUserProfile.pictureUrl || DEFAULT_PROFILE_PIC}
+                  src={pictureUrl || DEFAULT_PROFILE_PIC}
```

Comment:
Should this be using `??`
```suggestion
                  src={pictureUrl ?? DEFAULT_PROFILE_PIC}
```